CRM/Admin/Form/PaymentProcessor.php

```diff
@@ -43,6 +43,16 @@ class CRM_Admin_Form_PaymentProcessor extends CRM_Admin_Form {
 
   protected $_ppDAO;
 
+  /**
+   * @var int $_ppType Payment processor Type ID
+   */
+  protected $_ppType;
```

Comment:
Can we keep away from vars that are not readable - ie. ppType would be more readable as paymentProcessorType or processorType